frontend/public/components/catalog/catalog-items.tsx

```diff
@@ -175,7 +198,17 @@ export const groupItems = (items, groupBy) => {
   return items;
 };
 
-export class CatalogTileViewPage extends React.Component {
+export type CatalogTileViewPageProps = {
+  items: any;
```

Comment:
no type ? in the jsx file it was declared as an array

Reply:
I had left a TODO: remark that I am left to update the typings. I've updated the typings in all places possible. Please re-review :)

Reply:
Sorry, I didn't notice your TODO remark below the GIF yesterday :/